Gruntfile.js

```diff
@@ -166,6 +166,14 @@ module.exports = function (grunt) {
       }
     },
 
+    // ESLint is a replacement for JSHint + includes style linting + is extensible
+    eslint: {
+        target: [
+          'Gruntfile.js',
+          '<%= yeoman.app %>/scripts/{,*/}*.js'
+        ]
+    },
```

Comment:
Yeah I'm not sure there either.  Surely others have adopted a style guide on an already mature product, I'll do some research.